BookingSystem/Frontend/src/shared/components/Pagination.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from "react";
+
+class Pagination extends Component {
+  getPages(totalPages, currentPage, pageSize) {
```

Comment:
the common practice is to return a total number of items, but not pages. For example, below you set page size to 10 if it is not specified in arguments, but the totalPages might have been calculated for the different page size